test/serializers/attribute_test.rb

```diff
@@ -137,6 +137,33 @@ def false
         end
       end
 
+      def test_multiple_conditional_attributes_on_one_key
```

Comment:
Sure, what would you call the two tests?  Since really we're just testing the order of the attribute declarations